.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     outputs:
       changes: ${{ steps.changes.outputs.changes }}
     steps:
-    - uses: AurorNZ/paths-filter@v3
+    - uses: AurorNZ/paths-filter@v4
```

Comment:
Curious, would this be picked up by the bot?

Reply:
I have no idea! Want to wait on it and see?

Reply:
Probably not going to be covered by the bot https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates.